secboot/secboot_sb_test.go

```diff
@@ -33,7 +33,11 @@ import (
 	"path/filepath"
 
 	"github.com/canonical/go-tpm2"
+	"github.com/canonical/go-tpm2/linux"
+
 	sb "github.com/snapcore/secboot"
+	sb_efi "github.com/snapcore/secboot/efi"
+	sb_tpm2 "github.com/snapcore/secboot/tpm2"
 	. "gopkg.in/check.v1"
```

Comment:
```suggestion
	"github.com/canonical/go-tpm2/linux"
	. "gopkg.in/check.v1"

	sb "github.com/snapcore/secboot"
	sb_efi "github.com/snapcore/secboot/efi"
	sb_tpm2 "github.com/snapcore/secboot/tpm2"
```

(and then I think the imports need to be re-order alphabetically as well, but the grouping is stdlib, external, non-snapd repos, then snapd internal packages)